client/cypress/integration/add-product.spec.ts

```diff
@@ -11,7 +11,7 @@ const testProductToAdd: Product = {
   lifespan: 4,
   location: 'Florida',
   notes: 'A tropical fruit',
-  store: 'Pom de Terre',
+  store: 'Pomme de Terre',
```

Comment:
Thanks for fixing that. We should probably do a global search to make sure there aren't other things like this.